parity-crypto/src/lib.rs

```diff
@@ -24,9 +24,11 @@ extern crate scrypt as rscrypt;
 extern crate ripemd160 as rripemd160;
 extern crate sha2 as rsha2;
 extern crate digest as rdigest;
+extern crate hmac as rhmac;
```

Comment:
dumb q: What's up with all these `r` prefixes? "Rust"? Or is it temporary just while we have `ring` in the crate still?

Reply:
It's because there's a module named `hmac` in the crate. Having just `extern crate hmac;` triggers a compilation error. Switching to edition 2018 could fix this.

Reply:
> It's because there's a module named hmac in the crate.

Got it, so that explains `rdigest` as well. What about `rsha2`?

> Switching to edition 2018 could fix this.

I for one would prefer it if we did.

Reply:
That's out of scope of this PR though

Reply:
True, I'm just worried that we'll be here in ten years making conjectures about the secret meaning of the "r"-prefixes. Leave a TODO?